package.json

```diff
@@ -13,22 +13,28 @@
     "url": "https://github.com/aurelia/cli/issues"
   },
   "bin": {
-    "aurelia": "bin/aurelia-cli.js",
-    "au": "bin/aurelia-cli.js"
+    "aurelia": "dist/bin/aurelia-cli.js",
+    "au": "dist/bin/aurelia-cli.js"
   },
   "engines": {
     "node": ">=8.9.0"
   },
   "scripts": {
+    "build": "npm run build:ts && npm run build:dts && npm run build:copy-files",
+    "build:ts": "./node_modules/.bin/tsc",
```

Comment:
npm auto uses local `node_modules` first, so probably we don't need to be this specific? Besides that, if someone chose to install with any name that is not `node_modules`, this would break 😓

Reply:
pretty sure `"build:ts": "tsc"` does exact same thing.

Reply:
My primary reason for being specific in these paths is that this enforces using the project installed packages. I might be mistaken but I think node looks for globally installed packages first and then starts looking from the local dir up for node_modules folders.

Secondary I have had numerous situations where not specifically pointing to node modules failed to make chained commands work. I just tried to remove all from the commands and even though I would also expect it to work it does not, copy files for instance fails:
```cli
'copyfiles' is not recognized as an internal or external command,
operable program or batch file.
```
Unless I start using globally installed modules I don't think this will work without being this specific. 

@bigopon I just searched a bit on changing the dir, so far I only see the ability to change the prefix for globally installed modules. Local installation using `npm i` and `npm ci` will always end up in `node_modules`. So this setup would be just fine if someone would decide to change the global install.

Reply:
I think npm script will enforce local package for you. Same as you run `npx tsc` manually in console.

Reply:
@michaelw85
Here's how it works:
- Given a script defined in `package.json`: `"my-script": "foo"` if you call `npm run my-script`, npm will try to call `node_modules/.bin/foo` inside the local package. That is the convention - npm does not deviate from that unless you specifically give it the `--global` flag.
- npm itself runs globally, so it uses the current working dir to figure out its scope. If any of the scripts change the current working directory, npm can lose its scope and fall back to the global.
- An absolute path is no guarantee that it will work either. That will depend on whether the resolved path has already been cached or not - not something you want to rely on.

So, in this case. `"build:ts": "./node_modules/.bin/tsc",` doesn't do any harm but it will not do any good either. npm literally translates `"build:ts": "tsc",` to that: https://github.com/npm/cli/blob/latest/lib/npm.js#L381

Reply:
@fkleuver Could you checkout my branch, delete the `./node_modules` entries from the json and tell me if copy_files works for you when using the build task. With your explanation it should work flawlessly (which I was also expecting) but it does not.

Reply:
@michaelw85 
Here's what I did:
```
git clone https://github.com/michaelw85/cli.git aurelia-cli-2
cd aurelia-cli-2
git checkout initial_ts_setup
npm ci
```

Then I opened `package.json` and did a find/replace of `./node_modules/.bin/` with empty string. 5 occurrences replaced.

Then I ran `npm run build` and by the looks of it all succeeded:
![image](https://user-images.githubusercontent.com/9655331/61217589-4de9e400-a710-11e9-878d-7120cc611c78.png)

I do not have `copyfiles` installed globally. I am running node `12` if it matters.

Reply:
That is interesting, it did not work for me. I'm running node 10.15.1
I will give it a try on my own device when I get home if it fails I will upgrade and try again.

Reply:
@fkleuver I tried again, deleted node_modules, npm ci, replace paths and run build and it works fine.

@bigopon @3cp Removed './node_modules/...'